### DISCUSSION TITLE
GitHub CoPilot for Green Software Development

### DISCUSSION THREAD

**Discussion by balasomasundaram** [Green-Software-Foundation/discussions#115] [Ask Anything]
Do we have any plugins / support from GitHub Copilot for Green software development?

**Comment by Sophietn** [comment]
Hi @balasomasundaram, thanks for getting in touch with us. 

No, currently at the GSF we do not have any plugins / support from GitHub Copilot for Green software development. 

Do you have any examples or ideas about the sort of thing you'd find useful?
